File: laravel-demo/app/Http/Controllers/Pages/EducationController.php
<?php

namespace App\Http\Controllers\Pages;

use App\User;
use App\Http\Controllers\Controller;

class EducationController extends Controller
{
    public function coursesV1()
    {
        return view('pages.education.courses-v1', ['title' => 'Courses v1 - Pages']);
    }

    public function coursesV2()
    {
        return view('pages.education.courses-v2', ['title' => 'Courses v2 - Pages']);
    }

}

File: aspnet-mvc-demo/AspnetMvcDemo/Controllers/Pages/EducationController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;
using System.Web.Mvc;

namespace AspnetMvcDemo.Controllers.Pages
{
    public class EducationController : Controller
    {

         public ActionResult CoursesV1()
        {
            return View("~/Views/Pages/Education/CoursesV1.cshtml");
        }

         public ActionResult CoursesV2()
        {
            return View("~/Views/Pages/Education/CoursesV2.cshtml");
        }

    }
}
